### PR TITLE
new: add pomodoro app (fixed PR)

### PR DESCRIPTION
**Pull request by thiswillbeyourgithub**
(Clean PR after botching #320)

- new: add pomodoro app
- added pomodoro icon thank to user plan5
- refactor: ability to set multiple timers
- fix: forgot crediting author
- pomodoro: restart app when quitting and stopped
- moved pomodoro to apps directory
- various enhancement from authors fork

Sorry for the inconvenience!

I removed the code necessary to remember the latest values as #368  is not merged yet

### CODE REVIEW

File: apps/Pomodoro.py [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+"""
+
+import wasp
+import fonts
+import widgets
+import math
+import random
+from micropython import const
+
+# 2-bit RLE, 96x64, generated from png provided by plan5, 239 bytes
```
Comment (fgaz) [inline apps/Pomodoro.py:24]: The png is missing (other encoded icons reference the source png). Also it'd be best if this had proper attribution in the form of Co-Authored-By in the commit message

Comment (thiswillbeyourgithub) [reply apps/Pomodoro.py:24]: Unfortunately I don't think I have the original icon png anymore, just the screenshot of the running app.

Comment (fgaz) [reply apps/Pomodoro.py:24]: A screenshot should be fine as long as it encodes to the same RLE

**Comment by beardeddude** [comment]
As you may know after the merge of #403 the naming conventions and file placement have changed slightly. See https://wasp-os.readthedocs.io/en/latest/appguide.html#app-naming-conventions-and-placement for details.

**Comment by thiswillbeyourgithub** [comment]
Closed and moved to #418  as I couldn't add screenshots in my old branch

**Pull request closed without merging**